android/src/test/java/com/adyenreactnativesdk/configuration/CardConfigurationParserTest.kt

```diff
@@ -0,0 +1,142 @@
+package com.adyenreactnativesdk.configuration
+
+import android.util.Log
+import com.adyen.checkout.card.AddressConfiguration
+import com.adyen.checkout.card.KCPAuthVisibility
+import com.adyen.checkout.card.SocialSecurityNumberVisibility
+import com.adyen.checkout.card.data.CardType
+import com.facebook.react.bridge.ReadableArray
+import org.junit.Assert.*
+import org.junit.Test
+import org.mockito.MockedStatic
+import org.mockito.Mockito
+import org.mockito.Mockito.mock
+import org.mockito.Mockito.`when`
+
+class CardConfigurationParserTest {
+
+    private var mockStatic: MockedStatic<Log>? = null
+
+    @Test
+    fun testGetConfiguration() {
+    }
+
+    @Test
+    fun testGetBcmcConfiguration() {
+    }
+
+    @Test
+    fun testGetShowStorePaymentField() {
```

Comment:
We could think about using the "given when then" also in our test names. This could improve our test overview when errors are printed in the runners console. For example when we use the name "given_showStorePaymentFieldIsFalse_when_parseConfig_then_showStorePaymentFieldisFalse", then we can directly tell by reading the name of the tests what actually failed without opening the concrete test class. But let us discuss this first. Maybe that is applicable for v2.

Reply:
This one requires aligning with iOS.
Will address in another PR